packages/buidler-core/src/internal/sentry/anonymizer.ts

```diff
@@ -91,6 +91,12 @@ export class Anonymizer {
     };
   }
 
+  public anonymizeErrorMessage(errorMessage: string): string {
+    // the \\ before path.sep is necessary for this to work on windows
+    const pathRegex = new RegExp(`\\S+\\${path.sep}\\S+`, "g");
+    return errorMessage.replace(pathRegex, "<user-file>");
```

Comment:
Shouldn't this be replaceAll?

Reply:
The regex has the `g` modifier. Plus, `replaceAll` was added to V8 like, two weeks ago? :sweat_smile:

Reply:
Haha I didn't notice that when I Googled if it existed